src/services/githubService.js

```diff
@@ -180,4 +189,8 @@ const calcTopLanguage = userLanguagePercents => {
   return sortedLanguages[0] ? sortedLanguages[0].name : '';
 };
 
+const getStarGazers = async (repos) => {
```

Comment:
Is this really an async function?

Reply:
My bad. I am not fully aware of how async-await works. Since I don't have a http call inside, I think it need not be async, since it's not making a request within the function. This is my understanding. Shall I remove the async and await ?

Reply:
@lourdesr Yes that's exactly what I was thinking.